pycbc/inference/burn_in.py

```diff
@@ -553,6 +624,14 @@ def nacl(self, filename):
         # add the status for each parameter as additional information
         self.test_aux_info[test] = is_burned_in
 
+    def chain_moments(self):
+        """Applies estimate_sigma test"""
+        test = 'chain_moments'
+        sigmas,burn_in_iter,is_burned_in = chain_moments(self.sampler)
+        self.test_aux_info[test] = is_burned_in
```

Comment:
`is_burned_in` should be saved to `self.test_is_burned_in[test]`, not `test_aux_info`

Reply:
If you want to save the sigmas, store them to `test_aux_info`.